### PR TITLE
Creating CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by rsm23**
Creating CONTRIBUTING.md file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+
+
+
+## Getting Started
+
+0. [Fork the repo!](https://github.com/rsm23/ncs-jss.github.io/new/master#fork-destination-box)
```
Comment (ankitjain28may) [inline CONTRIBUTING.md:16]: Repo link should be the link of this repo rather than the link of your fork repo

**Review by ankitjain28may — CHANGES_REQUESTED** [review]

**Comment by rsm23** [comment]
Yeah my bad. :grinning:

**Comment by ankitjain28may** [comment]
@rsm23 Are you going to do this ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+0. Clone your fork of the repo: `git clone https://github.com/rsm23/ncs-jss.github.io.git`.
+0. Navigate to the project: `cd ncs-jss`.
```
Comment (ankitjain28may) [inline CONTRIBUTING.md:18]: It should be `cd ncs-jss.github.io`

**Review by ankitjain28may — CHANGES_REQUESTED** [review]

**Pull request closed without merging**